frontend/plan/components/schedule/Schedule.tsx

```diff
@@ -142,7 +183,20 @@ class Schedule extends Component {
             // for every conflict of size k, make the meetings in that conflict
             // take up (100/k) % of the square, and use `left` to place them
             // next to each other.
-            const group = Array.from(conflict.values());
+            const group: {
+                day: Day;
```

Comment:
This type should probably have a name -- maybe `MeetingBlock` or something like that?

Reply:
Extracted this type and placed it in `types.ts` under `MeetingBlock`